README.md

```diff
@@ -14,10 +18,41 @@ connections and responses from end endpoint.
 
 (TODO: some words to go along with these diagrams)
 
-#### Development setup
+### Development setup
+
+#### Node.js
+
+##### The requirements for running this application are Node.js 4.4.4 and npm 3.8.9.
+You should use Node Version Manager (nvm) to manage the versions of node.js on your local machine.
+To install please visit: https://github.com/creationix/nvm
+_If you are a on a mac and use [homebrew](http://brew.sh/) you can install nvm by typing: brew update && brew install nvm_
+
+Once you have nvm installed you should now install node.js version 4.4.4 run: 
+
+```bash
+nvm install 4.4.4
 ```
-git clone https://github.com/department-of-veterans-affairs/healthcare-application.git
-cd healthcare-application
+
+Once you have version node.js 4.4.4 installed install npm version 3.8.9 by running:
```

Comment:
Switch `node.js` and `version` so it reads: `Once you have node.js version 4.4.4`